curator/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN if [ -n "${LOCAL_REPO}" ] ; then \
     fi
 
 RUN yum install -y --setopt=skip_missing_names_on_install=False --setopt=tsflags=nodocs \
-        elastic-curator-${CURATOR_VER} python2-ruamel-yaml "python-elasticsearch < 5.5" \
```

Comment:
This syntax `"python-elasticsearch < 5.5"` works with `yum install`?

Reply:
yep! has for months :) but we can get rid of it now